runtimes/java-11.0/src/main/java/io/openruntimes/java/RuntimeRequest.java

```diff
@@ -0,0 +1,63 @@
+package io.openruntimes.java;
+
+import org.rapidoid.http.Req;
+
+import java.util.Map;
+
+public class RuntimeRequest {
+    private String payload;
+    private Map<String, String> headers;
+    private Map<String, String> env;
+
+    public RuntimeRequest(
+            String payload,
+            Map<String, String> headers,
+            Map<String, String> env
+    ) {
+        this.payload = payload;
+        this.headers = headers;
+        this.env = env;
+    }
+
+    public RuntimeRequest(Req request) {
+        Map<String, Object> data = request.data();
+        if (data.containsKey("payload")) {
+            this.payload = (String) data.get("payload");
```

Comment:
The default payload must be an empty string as you can see here. So please verify that once

https://github.com/open-runtimes/open-runtimes/blob/3261573d3a02e7dbad1868e465d2858a3a183494/runtimes/deno-1.13/server.ts#L21

These changes are currently in the 0.14 branch